src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1116,7 +1116,12 @@ internal SingleLookupResult CheckViability(Symbol symbol, int arity, LookupOptio
                 ? ((AliasSymbol)symbol).GetAliasTarget(basesBeingResolved)
                 : symbol;
 
-            if (WrongArity(symbol, arity, diagnose, options, out diagInfo))
+            // Check for external symbols marked with 'Microsoft.CodeAnalysis.Embedded' attribute
+            if (!this.Compilation.SourceModule.Equals(unwrappedSymbol.ContainingModule) && unwrappedSymbol.IsHiddenByCodeAnalysisEmbeddedAttribute())
```

Comment:
Could you point me to a test that demonstrates significance of the first condition?

---
In reply to: [114024707](https://github.com/dotnet/roslyn/pull/18715#discussion_r114024707) [](ancestors = 114024707,113549567,113533730)